tests/contrib/actions/reports/test_reports.py

```diff
@@ -0,0 +1,40 @@
+try:
+    from unittest.mock import MagicMock, patch, call, mock_open
+except ImportError:
+    from mock import MagicMock, patch, call, mock_open
+import pytest
+from unittest import TestCase
+
+from spidermon.contrib.actions.reports import CreateReport
+from spidermon.exceptions import NotConfigured
+
+TEMPLATE = "template"
+FILENAME = "filename"
+
+
+class CreateReportTests(TestCase):
```

Comment:
Sometime it could be useful, but to keep consistency over the other tests, usually we don't have a `TestCase` class for our tests.